modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -466,7 +466,8 @@ public static void removeBinding(Binding binding) throws AndesException {
      * @param subscription subscription
      * @throws AndesException
      */
-    private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue, Subscription subscription) throws AndesException, SubscriptionAlreadyExistsException {
+    private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue,
+                                                                   Subscription subscription) throws SubscriptionAlreadyExistsException, AMQException, AndesException {
```

Comment:
Do we need AndesException in this list now?

What about these locations? Can we change AndesException to AMQException.

https://github.com/wso2/andes/blob/master/modules/andes-core/broker/src/main/java/org/wso2/andes/server/queue/AMQQueue.java#L50

https://github.com/wso2/andes/blob/master/modules/andes-core/broker/src/main/java/org/wso2/andes/server/queue/SimpleAMQQueue.java#L541

They were added in https://github.com/wso2/andes/pull/433